src/components/Info/Info.style.js

```diff
@@ -1,13 +1,52 @@
 import styled from 'styled-components';
 
-const InfoWrapper = styled.section`
+const InfoWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  min-height: 348px;
+  max-width: 738px;
+  margin: 130px 0 100px 0;
```

Comment:
I  guess you use the min-height here to the spacing between the info sections, right? This is not optimal. If this was a real-world project somebody might come and want another section added for more information. The ideal would be that you just drop in another InfoSection in the Info component.

You can achieve that by using the space between the "How it works" and "About" sections as margin-top for the InfoSection component (106px). Then the top margin of this InfoWrapper needs to be changed to 24px